flake8_quotes/__init__.py

```diff
@@ -44,19 +44,27 @@ def __init__(self, tree, filename='(none)', builtins=None):
     def add_options(cls, parser):
         parser.add_option('--quotes', default='\'', action='store',
                           help='Quote to expect in all files (default: \')')
-        parser.add_option('--multiline-quotes', default='\'', action='store',
-                          help='Multiline quote to expect in all files (default: \')')
+        parser.add_option('--multiline-quotes', default='', action='store',
+                          help='Multiline quote to expect in all files (default: any)')
         parser.config_options.extend(['quotes', 'multiline_quotes'])
 
     @classmethod
     def parse_options(cls, options):
+        multiline_ignored = True
+
         if hasattr(options, 'multiline_quotes'):
             multiline_quotes = options.multiline_quotes
+
+            if len(multiline_quotes) == 0:
+                multiline_quotes = options.quotes
+            else:
+                multiline_ignored = False
```

Comment:
I did it this way as we need a reference to all of the possible `multiline_quotes` so that we can check if the current token is a multiline quote and we can ignore it if needed.